.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+_Please use this to only for bug reports. For questions or when you need help, you can use the [GitHub Discussions](https://github.com/kroxylicious/design/discussions)
```

Comment:
should there be a second `_` somewhere maybe after `reports`?

Reply:
Is design discussions the right place?   I would have thought a question discussion in this repo.

Reply:
Added the other `_` and pointed both this and the issue config at `https://github.com/kroxylicious/kroxylicious/discussions`